src/main/java/org/cryptomator/cryptofs/health/dirid/EmptyDirFile.java

```diff
@@ -0,0 +1,49 @@
+package org.cryptomator.cryptofs.health.dirid;
+
+import org.cryptomator.cryptofs.health.api.DiagnosticResult;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.cryptomator.cryptofs.health.api.CommonDetailKeys.DIR_ID_FILE;
+
+/**
+ * A critical diagnostic result of an empty directory id file (dir.c9r).
+ * <p>
+ * Even thou the empty directory ID exists, it is reserved for the root node of the crypto filesystem only.
+ * DUe to its nature, the root node has no corresponding dir.c9r file.
```

Comment:
```suggestion
 * Even though the empty directory ID exists, it is reserved for the root node of the crypto filesystem only.
 * Due to its nature, the root node has no corresponding dir.c9r file.
```

Please no slang outside of Slack